prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-overview/enrollee-overview.component.html

```diff
@@ -61,7 +61,7 @@ <h3>Enrollee Information</h3>
           <app-enrollee-review *ngIf="enrolment"
                                [enrolment]="enrolment"
                                [admin]="true"
-                               (route)="onRoute($event)">
+                               (route)="onRoute($event, true)">
```

Comment:
This means on every use of the route emitter within `EnrolleeReview` in admin you will always be redirecting to site registration.  Is that your intent?

If it is and there is only a single route within `EnrolleeReview` that gets emitted in the admin you can be more explicit and rename `onRoute` to `onWhateverYouNeedToDo()` to emphasize it has a singular use in admin and then you don't need any parameters.  Then you can refactor `onNavigateEnrollee` to use `this.routeUtils` directly instead since there isn't much benefit to it calling `onRoute`.

Reply:
Great point Marty. After some investigation, I found this breaks the routing for the buttons in `EnrolleeReview` when `showEditRedirect = true`, so this implementation has to change.

Reply:
doesnt look like `showEditRedirect` can be `true` in admin, as admins dont edit enrollees information

Reply:
refactored initially, may want to change the way the url is built